### ISSUE TITLE
Moes thermostat TS0601 - (_TZE200_ye5jkfsb) showing incorrect room temperature

### ISSUE THREAD

**Issue by welo78** [dresden-elektronik/deconz-rest-plugin#8042]
### Does the issue really belong here?

- [X] I definitively want to report a bug within deCONZ or its REST-API

### Is there already an existing issue for this?

- [X] I have searched the existing issues and there is none for the bug at hand

### Describe the bug

Hello, it seems that a Moes electric heating thermostat has two issues.

1. Temperature is incorrectly displayed as 0 Celsius (or wrongly parsed/calculated) or 2 degrees when temperature is 22 in the room
2. Home assistant picks it up as increments of 0,5 degree when it can only make 1 degree steps

Thermostat in question: https://www.aliexpress.com/item/1005005963529004.html?spm=a2g0o.order_list.order_list_main.11.406c1802Nirlz9

### Steps to reproduce the behavior

1. Log into Conbee gateway
2. Check thermostat tab
3. Thermostat showing incorrect temperature

### Expected behavior

1. Display correct room temperature
2. Make increments of 1 degree step instead of 0,5 in home assistant (can be fixed in HA)

### Screenshots

![image](https://github.com/user-attachments/assets/faa91054-4a25-460e-9e19-5d3e7cc0e432)

### Environment

- Host system: Raspberry Pi 4
- Running method: Raspbian + Home Assistent deCONZ Add-on
- Firmware version:26400500
- deCONZ version: 2.28.1
- Device: RaspBee I 
- Is there any other USB or serial devices connected to the host system? If so: Z-Wave dongle

### deCONZ Logs

2024-11-27 09:51:39.666 DEBUG (MainThread) [pydeconz.websocket] {"attr":{"id":"23","lastannounced":null,"lastseen":"2024-11-27T08:50Z","manufacturername":"_TZE200_ye5jkfsb","mode":1,"modelid":"TS0601","name":"Thermostat 23","swversion":"1.0.1","type":"ZHAThermostat","uniqueid":"58:3b:c2:ff:fe:90:22:8b-01-ef00"},"e":"changed","id":"23","r":"sensors","t":"event","uniqueid":"58:3b:c2:ff:fe:90:22:8b-01-ef00"}
2024-11-27 09:51:40.797 DEBUG (MainThread) [pydeconz.websocket] {"attr":{"id":"23","lastannounced":null,"lastseen":"2024-11-27T08:51Z","manufacturername":"_TZE200_ye5jkfsb","mode":1,"modelid":"TS0601","name":"Thermostat 23","swversion":"1.0.1","type":"ZHAThermostat","uniqueid":"58:3b:c2:ff:fe:90:22:8b-01-ef00"},"e":"changed","id":"23","r":"sensors","t":"event","uniqueid":"58:3b:c2:ff:fe:90:22:8b-01-ef00"}

### Additional context

I am all up for testing any DDF if this is a unsupported device at the moment. However as it picked it up I kinda thought it is supported.

**Comment by Smanar** [comment]
Perhaps https://github.com/Koenkk/zigbee2mqtt/issues/11980 ?

But it seem working for somes users, but not all.

We can try editing the DDF devices\moes\Moes_TZE200_ztvwu4nk_thermostat.json
How many TRV have you  ? because editing direclty this file will impact ALL yours devices using the same DDF

And can you check the value returned by your device ?

It's visible on logs in Deconz/help/Debug view, I think the flag "DDF" will be enought to see the value used by the device.

**Comment by welo78** [comment]
Hello @Smanar thank you for the reply. I added the debug message to the original post.

I have 1 of this device for testing purposes. I am looking for swapping non-smart thermostat for zigbee operated.

As for DDF when I enabled debug view and ticked DDF it started dumping all kinds of DDFs it knows in a long list. Will this suffice?
![image](https://github.com/user-attachments/assets/d3f2de01-52ae-42bf-9097-5a15a8a09f64)
